packages/main/src/RPA/Database.py

```diff
@@ -1,23 +1,527 @@
+import importlib
 import logging
-from DatabaseLibrary import DatabaseLibrary
+import sys
 
 from RPA.Tables import Table
 
+try:
+    import ConfigParser
+except ImportError:
+    import configparser as ConfigParser
 
-class Database(DatabaseLibrary):
-    """Library handling different database operations.
 
-    Extends functionality of DatabaseLibrary, for more information see
-    https://github.com/franz-see/Robotframework-Database-Library
-    """
+class Configuration:
+    """Class to handle configuration from config files
+    and class init"""
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self):
+        self.configuration = {}
+        self.module_name = None
+
+    def parse_arguments(
+        self,
+        module_name,
+        database,
+        username,
+        password,
+        host,
+        port,
+        charset,
+        config_file: str,
+    ):
+        config = ConfigParser.ConfigParser()
+        config.read([config_file])
+
+        self.configuration = {}
+        self.module_name = module_name or (
```

Comment:
The configparser api provides the option for fallbacks, so it looks like you could write this pattern like this:
`self.module_name = module_name or config.get("default", "module_name", fallback=None)`